examples/closures/src/lib.rs

```diff
@@ -74,7 +74,7 @@ fn setup_clock(window: &Window, document: &Document) -> Result<(), JsValue> {
 
     // The instances of `Closure` that we created will invalidate their
     // corresponding JS callback whenever they're dropped, so if we were to
-    // normally return from `run` then both of our registered closures will
+    // normally return from `setup_clock` then both of our registered closures will
     // raise exceptions when invoked.
```

Comment:
I think the comment is from the first version of the example when everything was put in a single `fn run`. Now it is split into several `fn`s, and you want to fix them, then I think other places should be fixed too. The comment is now applied to a single instance of Closure `a` at L66, then things should be fixed, e.g.: "instances" (become "instance"), "their", "they"... and finally add a note to `setup_clicker` (below) say that people may want to see why `a.forget()` as notes in `setup_clock` above.

Reply:
@limira I added the changes you suggested. I was not sure about whether to change "exceptions" to "exception". I am assuming that a single closure can throw at most one exception?

Reply:
To me it's good enough. But others may provide a more precise comments.